lib/helpers.ts

```diff
@@ -143,18 +145,26 @@ export function computeDashboardData(files: Entity[]): EntityReport[] {
     const uniqueOrgans = new Set();
     const uniqueBiospecs = new Set();
     const uniqueCases = new Set();
+
     for (const file of files) {
         if (file.atlasid) {
             uniqueAtlases.add(file.atlasid);
         }
-        for (const biospec of file.biospecimen) {
-            uniqueBiospecs.add(biospec.BiospecimenID);
+        for (const biospecimen of file.biospecimen) {
+            uniqueBiospecs.add(biospecimen.BiospecimenID);
+        }
+        for (const diagnosis of file.diagnosis) {
+            uniqueCases.add(diagnosis.ParticipantID);
+            uniqueOrgans.add(getNormalizedOrgan(diagnosis));
         }
-        for (const diag of file.diagnosis) {
-            uniqueCases.add(diag.ParticipantID);
-            uniqueOrgans.add(diag.TissueorOrganofOrigin);
```

Comment:
@onursumer so if i understand the `diag.TissueorOrganofOrigin` field is incorrect? What's the diff between using `getNormalizedOrgan`?